apps/studio/src/pages/home/SideBar.tsx

```diff
@@ -183,6 +187,12 @@ export const SideBar: FunctionComponent<SideBarProps> = ({
         </Box>
       </Box>
 
+      { webStudioArtistReferralCampaign && (
+        <Box bottom={ 0 } mb={ 2 } mt="auto" mx={ -1 } position="sticky" zIndex={ 2 }>
```

Comment:
This was the design aesthetic provided on Figma, but I'm not sold on it. It floats and overlays in the bottom corner until you scroll to the bottom where it will end up. Can demo it in mobile or in desktop by increasing zoom of page and scrolling. However, at a normal zoom, desktop view, it doesn't make much sense. Will leave for now, but will let product/design know about the need for a better banner UX that doesn't also hide the sidebar links.